pkg/cmd/gist/edit/edit.go

```diff
@@ -108,6 +108,9 @@ func editRun(opts *EditOptions) error {
 	if gistID == "" {
 		cs := opts.IO.ColorScheme()
 		if gistID == "" {
+			if !opts.IO.CanPrompt() {
+				return cmdutil.FlagErrorf("gist ID or URL required when not running interactively")
+			}
```

Comment:
Nit: I think we should keep the spacing consistent between the changes to `gist edit` and `gist view` 😄  

```suggestion
			if !opts.IO.CanPrompt() {
				return cmdutil.FlagErrorf("gist ID or URL required when not running interactively")
			}
			
```